cmake/Modules/common.cmake

```diff
@@ -11,12 +11,10 @@ include_directories(
 link_directories(${GTEST_LIBRARIES} ${COMPUTECPP_LIBS})
 
 link_libraries(
-  #opencv libs
-  ${OpenCV_LIBS}
   #gtest libs
   ${GTEST_BOTH_LIBRARIES}
+  #opencv libs
+  ${OpenCV_LIBS}
   #sycl libs
   ${COMPUTECPP_RUNTIME_LIBRARY}
-  #others
-  -lpthread
```

Comment:
is pthread no longer needed?

Reply:
CMake 3.1.0+ is able to do just the following:
In your root CMakeLists.txt
- find_package(Threads REQUIRED)
then, in the examples.cmake: after your add_executable(...) bit,
you target the lib as follows:
- target_link_libraries(my_app Threads::Threads)